schema/buildings/building.yaml

```diff
@@ -25,27 +25,123 @@ properties:
       - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
       - "$ref": ./defs.yaml#/shapeContainer
     properties:
-      class:
+      subtype:
         description: >-
           A broad category of the building type/purpose. When the
           current use of the building does not match the built purpose,
-          the class should be set to represent the current use of the
+          the subtype should be set to represent the current use of the
           building.
         type: string
         enum:
-          - residential
-          - outbuilding
           - agricultural
+          - civic
           - commercial
-          - industrial
           - education
-          - service
+          - entertainment
+          - industrial
+          - medical
+          - military
+          - outbuilding
+          - parking
           - religious
-          - civic
+          - residential
+          - service
           - transportation
-          - medical
-          - entertainment
+      class:
+        description: >-
+          Further delineation of the building's built purpose.
+        type: string
+        enum:
```

Comment:
Does this list also include values from `amenity`, `turism` or other tags that are used in mapping building to its `subtype`?

Reply:
No, it _only_ takes values from `building` tag.